data/timesketch.conf

```diff
@@ -380,11 +380,17 @@ LLM_PROVIDER_CONFIGS = {
         },
     },
     'llm_summarize': {
-        'aistudio': {
-            'model': 'gemini-2.0-flash-exp',
+        'vertexai': {
+            'model': 'gemini-2.0-flash-001',
             'project_id': '',
```

Comment:
Is changing this default intended?

Reply:
Yes, as the flash model has gone to GA since and is no longer experimental